LICENSE

```diff
@@ -178,15 +179,15 @@
    APPENDIX: How to apply the Apache License to your work.
 
       To apply the Apache License to your work, attach the following
-      boilerplate notice, with the fields enclosed by brackets "[]"
+      boilerplate notice, with the fields enclosed by brackets "{}"
       replaced with your own identifying information. (Don't include
       the brackets!)  The text should be enclosed in the appropriate
       comment syntax for the file format. We also recommend that a
       file or class name and description of purpose be included on the
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright 2016-2019 Istio Authors
```

Comment:
Note the CNCF legal eagles have posted a nice MD doc describing how copyrights should be handled.  I agree with everything in the document.  I'd recommend taking a look here:

https://github.com/cncf/foundation/blob/master/copyright-notices.md

Reply:
See https://github.com/istio/common-files/pull/34 for an update that removes the date range.